src/main/java/chess/service/ChessRoomService.java

```diff
@@ -111,15 +114,16 @@ private static String createRightAnnouncement(State state) {
 		return Announcement.ofEnd().getString();
 	}
 
-	public String loadAnnouncementMessage(int roomId) throws SQLException {
-		return announcementDao.findAnnouncementByRoomId(roomId).getMessage();
+	public String loadAnnouncementMessage(int roomId) {
+		return announcementRepository.findByRoomId(roomId).orElseThrow(DaoNoneSelectedException::new)
+				.getMessage();
 	}
 
-	public int saveNewAnnouncementMessage(int roomId) throws SQLException {
-		return announcementDao.addAnnouncement(Announcement.ofFirst().getString(), roomId);
+	public void saveNewAnnouncementMessage(int roomId) {
+		announcementRepository.save(Announcement.ofFirst().getString(), roomId);
 	}
```

Comment:
```suggestion
	public void saveNewAnnouncementMessage(int roomId) {
		AnnouncementDto announcementDto = new AnnouncementDto(Announcement.ofFirst().getString(), roomId);
		announcementRepository.save(announcementDto);
	}
```

이렇게 entity 객체를 바로 넣으면 쿼리문 만들 필요없이 crudRepository가 지원해주는 기본 save 메서드를 사용할 수 있다고하네요! 
참고해보세요~

Reply:
오.. 친절히 알려주셔서 감사합니다..! 👍